Library/Formula/cryptol.rb

```diff
@@ -0,0 +1,34 @@
+require "language/haskell"
+
+class Cryptol < Formula
+  include Language::Haskell::Cabal
+
+  homepage "http://www.cryptol.net/"
+  url "https://github.com/GaloisInc/cryptol.git", :tag => "v2.2.1"
```

Comment:
Would a revision be redundant with the tag? The Formula cookbook makes them sound like either/or: "To use a specific commit, tag, or branch from a repository, specify head with the :revision, :tag, or :branch option, like so..."